src/bin/example-metadata.rs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+extern crate parquet;
+
+use std::fs::File;
+use std::path::Path;
+
+use parquet::file::metadata::{FileMetaData, ParquetMetaData};
+use parquet::file::reader::{FileReader, SerializedFileReader};
+
+// Run example
+// ===
+// $ cargo run --bin example-metadata
+//
+// Console output
+// ===
+// Number of row groups: 1
+// Version: 1
+// Created by: Some("parquet-mr version 1.8.1 (build 4aba4dae7bb0d4edbcf7923ae1339f28fd3f7fcf)")
+// Number of rows: 4
+// Column INT32: "a"
+// Column INT64: "b"
+// Column BYTE_ARRAY: "c"
+// Column INT32: "d.a"
+// Column INT64: "d.b"
+// Column BYTE_ARRAY: "d.c"
+// Column INT32: "e.list.element"
+//
+fn main() {
+  let path = Path::new("/Users/sadikovi/developer/parquet-rs/data/example.snappy.parquet");
```

Comment:
This is not gonna work. Maybe change to a relative path?

Reply:
I fixed it, thanks!